pyfolio/tears.py

```diff
@@ -925,12 +913,15 @@ def create_interesting_times_tear_sheet(
     bubble burst, EZB IR, Great Recession (August 2007, March and September
     of 2008, Q1 & Q2 2009), flash crash, April and October 2014.
 
+    benchmark_rets must be passed, as it is meaningless to analyze performance
```

Comment:
Not sure we should make that decision for the user, many events in there are not related to macro market events.

Reply:
So we should change the interesting times tear sheet to simply plot cumulative returns during interesting times? That should probably be a separate PR: I'll file an issue for that though.

Reply:
Filed under #538

Reply:
My read of the code suggested it was already possible to run it without a benchmark.

Reply:
You're right, not much work to do it! Closes #538

Reply:
You're right, not too much work to do it.